<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Worldview</class>
 <widget class="QWidget" name="Worldview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>373</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>539</width>
    <height>373</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>World View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="l_view">
     <property name="minimumSize">
      <size>
       <width>360</width>
       <height>180</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="c_mode">
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="placeholderText">
      <string>View Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="b_refresh">
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="l_avgheat">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Average heat:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="l_avgheight">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Average height:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
